nginx/Dockerfile

```diff
@@ -1,14 +1,15 @@
 FROM node:10 AS frontend
-COPY ./public/node /code
+COPY ./public/node/frontend/package.json /code/frontend/
 RUN npm install -g gulp
 
 WORKDIR /code/frontend
+RUN npm install
+COPY ./public/node/ /code/
 RUN wget http://www.fawe.de/volpiano/volpiano51_web.zip
 RUN unzip volpiano51_web.zip
 RUN rm volpiano51_web.zip
 RUN mv volpiano51_web/volpiano.woff ../static/fonts/volpiano.woff
-RUN npm install
-RUN gulp build --release
+RUN gulp build
```

Comment:
I'm curious about the removal of the `--release` flag in `gulp build`. I thought this made the difference between minified and not-minified `js`. We want minified by default, except when debugging, right?

Reply:
Also, can we remove the `npm install`?

Reply:
> Also, can we remove the `npm install`?

No, but I moved it to a different stage in the process...see new line 6.

Reply:
> I'm curious about the removal of the `--release` flag in `gulp build`. I thought this made the difference between minified and not-minified `js`. We want minified by default, except when debugging, right?

Ah, yes...that's an error...I was debugging when I committed and forgot to change it back. Will do now.